src/Hocon.Configuration/HoconConfigConverter.cs

```diff
@@ -4,20 +4,26 @@
 using System.Text;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Hocon
 {
     class HoconConfigConverter : JsonConverter<Config>
```

Comment:
Let's not do this - we don't want a Newtonsoft.Json dependency in HOCON if we can avoid it. Besides, isn't this a relatively small issue?

Reply:
Its a rather large issue, since a lot of things depends on serialization. I'll try to find a new solution for this.

Reply:
Fixed. Moved all serialization to `Akka.Serialization.NewtonSoftJsonSerializer`